CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Added
+
+- add option to reinstall `lean4-pr-releases` toolchains, ensuring CI runs with thev latest version
```

Comment:
```suggestion
- add option to always reinstall `lean4-pr-releases` toolchains, ensuring CI runs with the latest version

Reply:
Thanks! I'll open a new PR to fix this. :)

Reply:
#123